common/compositor/va/varenderer.cpp

```diff
@@ -292,7 +292,8 @@ bool VARenderer::Draw(const MediaState& state, NativeSurface* surface) {
   param_.filter_flags = filter_flags_;
 // currently rotation is only supported by VA on Android.
 #ifdef DISABLE_CURSOR_PLANE
```

Comment:
we should probably change this check to be #ifdef VA_MAJOR_VERSION > 1

Reply:
@xuguangxin Can u help here, how do we ensure we get the right libva versions which includes this params

Reply:
@kalyankondapally , why we need to keep libva 1.0 code here.  We will remove libva from AIA and AIA will get libva 2.0 based solution in following weeks.

Reply:
But till than we need to have the code compiling right ?